### PR TITLE
feat(generative_ai): add gemini pro basic and config examples

### PR DESCRIPTION
**Pull request by msampathkumar**
## Description

Add two Gemini pro vision samples

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/gemini_pro_basic_example.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    model = GenerativeModel(model_name="gemini-pro-vision")
+
+    # Load example image
+    image_url = (
+        "gs://generativeai-downloads/images/scones.jpg"
+    )
+    image_content = Part.from_uri(image_url, "image/jpeg")
+
+    # Query the model
+    response = model.generate_content(["what is recipe in image:", image_content])
```
Comment (gericdong) [inline generative_ai/gemini_pro_basic_example.py:33]: Suggest to put image first.

File: generative_ai/gemini_pro_config_example.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    image_content = Part.from_data(
+        data=base64.b64decode(encoded_image), mime_type="image/jpeg"
+    )
+
+    # Generation Config
+    config = {"max_output_tokens": 2048, "temperature": 0.4, "top_p": 1, "top_k": 32}
+
+    # Generate text
+    response = model.generate_content(
+        ["what is this image", image_content],
```
Comment (gericdong) [inline generative_ai/gemini_pro_config_example.py:41]: Suggest to put image first.

File: generative_ai/test_gemini_examples.py [hunk 3]
```diff
@@ -37,6 +39,33 @@ def test_gemini_guide_example() -> None:
...
+    #  download the image
+    fname = "scones.jpg"
+    url = "https://storage.googleapis.com/generativeai-downloads/images/scones.jpg"
+    urllib.request.urlretrieve(url, fname)
+
+    if os.path.isfile(fname):
+        text = gemini_pro_config_example.generate_text(PROJECT_ID, LOCATION)
+        text = text.lower()
+        assert len(text) > 0
+        assert "recipe" in text or "table" in text
```
Comment (rsamborski) [inline generative_ai/test_gemini_examples.py:61]: suggestion: the sample is asking to describe the picture (prompt: "what is in the image"), so probably the text will mention scones, cookies, coffee etc.

**Review by rsamborski — APPROVED** [review]
Approved with a couple nits.

**Review by gericdong — APPROVED** [review]

**Pull request merged**